### ISSUE TITLE
GitHub Actions: Adding missing labels to issues lacking the minimum 3

### ISSUE THREAD

**Issue by Aveline-art** [hackforla/website#2013]
### Overview
As a developer on the team, I need to know exactly what an issue entails so that I can quickly filter for one that is appropriate for my role and skill level.

### Scope

Our team currently requires every issue to have at minimum 3 labels: a size label, a feature (**or** p-feature) label, and a role label. This GHA will alert the team member, who created the issue, that at least one of the labels is missing.

### Psudo-code
- [x] Before starting, scroll through our labels to get an understanding of what this issue is about.
- [x] Trigger: on issue creation
  - [x] If the issue lacks one of the 3 labels:
    - [x] Add any of the 'Feature: missing', 'role: missing', or 'Size: missing' labels as appropriate.
    - [x] Post a comment on the issue alerting them that one of the three labels are missing.
    - [x] The comment needs to have
      - [x] an @username to the creator of the issue
      - [x] a list of the label types that are missing
      - [x] a reminder to also move their issue into the project board and the new issue creation column
      - [x] a link to GitHub's documentation on how to 1) add labels, 2) move an issue into a column
      - [x] instructions to remove the missing labels
  - [x] Else:
      - [x] Post a nice comment indicating that the assignee has done well with the labels
        - [x] The comment should also include a friendly reminder to also move their issue into the project board and the new issue creation column
- [x] In the interest of maintenance, the comment should exist as an md file outside of the code. This allows changing the comment without diving into the code itself. Any unstable variables should be noted with placeholders. See [here ](https://github.com/hackforla/website/blob/gh-pages/github-actions/pr-instructions/pr-instructions-template.md) for an example.

### Checks

- [x] When an issue is created with all 3 of the minimum labels: no comment is made.
- [x] When an issue is created with 'role: design' and 'size: small' and no other labels: a comment is made noting that a feature or p-feature is missing.
- [x] When an issue is edited with any number of labels (0-positive inf), no comment is made.
- [x] Your pull request contains at least 1 md file.

### Additional items

- [ ] If the status label wiki in #1983 or the issue wiki in #1941 is finished
  - [ ] Tack on links to the above mentioned wikis into the automatic comment made by this GHA.
- [x] Else, if either is not done
  - [x] Tack on additional action items to one or both issue as appropriate, noting that the automatic comment needs to include links to those wiki.

### Resources/Instructions
Never done GitHub actions? [Start here!](https://docs.github.com/en/actions)

Note that you might want to do something outside the scope of the above psudo-code. If so, be sure to leave comments in your PR or this issue that justifies your reasoning. If you feel you need guidance, be sure to reach out! We cannot foresee whether this issue is solvable, or what hard decisions have to be made, but we would love to hear and help you!

#### Additional resources:
[Events that trigger workflows](https://docs.github.com/en/actions/reference/events-that-trigger-workflows)
[Workflow syntax for GitHub Actions](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions)
[GitHub RESTAPI](https://docs.github.com/en/rest)
[GitHub GraphQL](https://docs.github.com/en/graphql)
